internal/server/audit/audit.go

```diff
@@ -142,10 +138,9 @@ func decodeToEvent(kvs []attribute.KeyValue) (*Event, error) {
 
 // Metadata holds information of what metadata an event will contain.
 type Metadata struct {
-	Type   Type   `json:"type"`
-	Action Action `json:"action"`
-	IP     string `json:"ip,omitempty"`
-	Actor  string `json:"actor,omitempty"`
+	Type   Type              `json:"type"`
+	Action Action            `json:"action"`
+	Actor  map[string]string `json:"actor,omitempty"`
```

Comment:
can we make this a type `Actor` that is a struct?

Reply:
That would be ideal, but the k/v for an Actor can be dynamic based on the auth method used.